GenderPayGap.WebUI/BackgroundJobs/ScheduledJobs/AnonymiseRetiredDuplicateUsersJob.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Linq;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Services;
+
+namespace GenderPayGap.WebUI.BackgroundJobs.ScheduledJobs
+{
+    public class AnonymiseRetiredDuplicateUsersJob
+    {
+
+        private readonly AuditLogger auditLogger;
+        private readonly IDataRepository dataRepository;
+
+        public AnonymiseRetiredDuplicateUsersJob(IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+
+        //Remove any duplicate retired user details
+        public void AnonymiseRetiredDuplicateUsers()
+        {
+            JobHelpers.RunAndLogJob(AnonymiseRetiredDuplicateUsersAction, nameof(AnonymiseRetiredDuplicateUsers));
+        }
+
+        private void AnonymiseRetiredDuplicateUsersAction()
+        {
+            List<User> users = dataRepository.GetAll<User>().ToList();
+
+            var usersWithMultipleAccounts = users
+                .Where(u => u.HasBeenAnonymised == false)
```

Comment:
what about `.Where(u => !u.HasBeenAnonymised)`

Reply:
Well this is embarrassing! :) Good catch